plugin/pkg/admission/security/podsecuritypolicy/admission.go

```diff
@@ -231,25 +226,34 @@ func (c *PodSecurityPolicyPlugin) computeSecurityContext(a admission.Attributes,
 		return nil, "", nil, fmt.Errorf("no providers available to validate pod request")
```

Comment:
the call to logProviders above is not useful now since the list is never filtered

Reply:
removed (kept the error logging)